cmd/plugins/juju-metadata/validateimagemetadata.go

```diff
@@ -136,8 +141,28 @@ func (oes *overrideEnvStream) Config() *config.Config {
 	return newCfg
 }
 
-func (c *validateImageMetadataCommand) Run(context *cmd.Context) error {
-	params, err := c.createLookupParams(context)
+func (c *validateImageMetadataCommand) Run(ctx *cmd.Context) error {
+	var (
+		base series.Base
+		err  error
+	)
+	// Note: we validated that both series and base cannot be specified in
+	// Init(), so it's safe to assume that only one of them is set here.
+	if c.series != "" {
+		ctx.Warningf("series flag is deprecated, use --base instead")
```

Comment:
s/--base/-b/

I'm undecided on "series", as the literal flag is "-s"

Reply:
Yeah, I don't like `-s` either.